css/properties/math-style.json

```diff
@@ -0,0 +1,67 @@
+{
+  "css": {
+    "properties": {
+      "math-style": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/math-style",
+          "support": {
+            "chrome": {
+              "version_added": "83",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Enable experimental Web Platform features"
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": false
```

Comment:
I don't see anything in the diff which suggests that this has been switched off for Android. Can you copy the data from chrome to chrome_android? Or show me how wrong I am?